parity/deprecated.rs

```diff
@@ -37,269 +39,5 @@ impl fmt::Display for Deprecated {
 pub fn find_deprecated(args: &Args) -> Vec<Deprecated> {
```

Comment:
It is probably worth commenting why we will end up with a empty function/module that doesn't do anything after this PR..

Reply:
Done